system-int-test/test/cmr/system_int_test/search/collection_with_new_granule_search_test.clj

```diff
@@ -297,7 +297,26 @@
           "Keyword"
           {:keyword "PROV1"} [coll-temporal-match coll-temporal-no-match
                               coll-spatial-match coll-w-june-2016-granule
-                              coll-w-may-2010-granule coll-platform-match])))))
+                              coll-w-may-2010-granule coll-platform-match]))
+
+      (testing "has_granules_created_at success in mutliple formats with sort"
+        (doseq [format [:atom :dif :dif10 :echo10 :iso19115 :json :opendata :xml :umm-json]]
```

Comment:
You can add a testing after the `doseq` as well so that when a test fails you know exactly which format failed.

```
(testing (str "format: " format)
...
```
The error report will be "has_granules_created_at success in mutliple formats with sort format :atom"

Reply:
For some reason I didn't even realize this was a possibility, very cool.